### PR TITLE
FIX Deprecation in RN 0.25

### PR DESCRIPTION
**Pull request by sarovin**

### CODE REVIEW

**Comment by cridenour** [comment]
At a quick glance, this looks good. Does this pass our new fancy eslint rules?

File: src/NavBar.js [hunk 1]
```diff
@@ -23,8 +23,10 @@
  * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
  */
-import React, {
+ import React, {
```
Comment (charpeni) [inline src/NavBar.js]: Useless space

File: src/Router.js [hunk 2]
```diff
@@ -6,10 +6,12 @@
  * LICENSE file in the root directory of this source tree.
  *
  */
-import React, {
+ import React, {
```
Comment (charpeni) [inline src/Router.js]: Useless space

**Comment by sarovin** [comment]
i am sorry ;)

**Comment by cridenour** [comment]
No need to apologize - thanks for helping :)

**Comment by charpeni** [comment]
Good job :)

LGTM if you can tell us if it pass the eslint rules ? I will setup CircleCI really soon, until that we will trust you.

**Comment by sarovin** [comment]
yes, it's pass the eslint rules ;)

**Pull request merged**